src/sim.py

```diff
@@ -57,6 +58,8 @@ def step(self):
                 elif type(car) == AutonomousCar:
                     autonomous_car_env = self.highway.get_autonomous_car_env(car_ind, lane_ind)
                     car.take_action(self.step_time, car_env, autonomous_car_env)
+                elif type(car) == Truck:
```

Comment:
isn't it the same function in both classes?

Reply:
or is there a difference between `car.take_action(self.step_time)` in case of normal car and truck

Reply:
No, a boolean is added to ` choose_speed` fun. When it is a Truck to limit the speed.